### ISSUE TITLE
Wrong description in iot_ble_numericComparison.c

### ISSUE THREAD

**Issue by jnz86** [aws/amazon-freertos#1820]
**Describe the bug**
`iot_ble_numericComparison.c`
```
* @file aws_ble_numericComparison.c
* @brief A simple OTA update example.
(continues wrong from there)
```
  
and  
  
`iot_ble_numericComparison.h`
```
/**
 * @file aws_iot_network.h
 * @brief Header file containing the network management APIs
 */
```

**Comment by leegeth** [comment]
@jnz86 

Thanks for bringing this issue to our attention. We will address this issue.

**Comment by leegeth** [comment]
We have addressed with PR #1826 . Closing the issue. Please open a new issue if there any other questions or issues.
